### DISCUSSION TITLE
FHIR - OPEN SOURCE

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5796] [Q&A] [answered]
Hello! I would like to know if there is any way to have FHIR as a connector in the open source version.

I appreciate any information.

**Comment by pacmano1** [comment, marked as answer]
FHIR is just API calls.  So that is http senders (or listeners).  So yes.

The commercial extension can make things easier, but it is not required.

**Reply by 5011-5011** [reply]
Thank you, do you have some kind of extra info that you can recommend me to see more specifically.

**Reply by pacmano1** [reply]
If you are needing to make FHIR calls or understand it more in general, I would use https://hapi.fhir.org/

**Reply by 5011-5011** [reply]
Thank you so much!
